src/Kernel.php

```diff
@@ -139,4 +137,49 @@ public function loadRoutes(LoaderInterface $loader): RouteCollection
 
         return $collection;
     }
+
+    private function loadMainBundles(string $path): array
+    {
+        return require $path;
+    }
+
+    private function loadAdditionalBundlesFromEnv(array &$contents): bool
```

Comment:
The `FromEnv` suffix is more appropriate to the next private message
```suggestion
    private function loadAdditionalBundles(array &$contents): bool
```